server/__init__.py

```diff
@@ -83,25 +81,20 @@ def run_lobby_server(
         games.clear_dirty()
         player_service.clear_dirty()
 
-        tasks = []
         if dirty_queues:
-            tasks.append(
-                ctx.broadcast({
-                        'command': 'matchmaker_info',
-                        'queues': [queue.to_dict() for queue in dirty_queues]
-                    },
-                    lambda lobby_conn: lobby_conn.authenticated
-                )
+            ctx.write_broadcast({
+                    'command': 'matchmaker_info',
+                    'queues': [queue.to_dict() for queue in dirty_queues]
+                },
+                lambda lobby_conn: lobby_conn.authenticated
```

Comment:
if we talk about performance, you can move that lambda out of the scope, to not to create this function every time

Reply:
Well you see I'm basing my performance improvements on actual data and I can tell you that refactoring this lambda function will make absolutely zero difference. In fact this is what lambda functions are for.

```
python3 -m timeit "lambda lobby_conn: lobby_conn.authenticated"
10000000 loops, best of 3: 0.0614 usec per loop
```

Reply:
Ok